File: Server/Models/AccountViewModel/UserDataViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Models.AccountViewModel
{
    public class UserDataViewModel
    {
        public string Id { get; set; }
        public string UserName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
    }
}

File: Client/src/app/models/UserDataViewModel.ts
export class UserDataViewModel{

    Id:string;
    UserName:string;
    FirstName: string;
    LastName: string;
    PhoneNumber: string;

    constructor(
        _Id:string,
        _UserName: string,
        _FirstName: string,
        _LastName: string,
        _PhoneNumber: string,
    ){
        this.Id = _Id;
        this.UserName = _UserName;
        this.FirstName = _FirstName;
        this.LastName = _LastName;
        this.PhoneNumber = _PhoneNumber;
    }
}